### PR TITLE
Add Java Example Project

### PR DESCRIPTION
**Pull request by mai93**
Add example java project to be used for manual testing.

### CODE REVIEW

File: examples/java/greetings_project/WORKSPACE [hunk 1]
```diff
@@ -0,0 +1,25 @@
+load("@bazel_tools//tools/build_defs/repo:jvm.bzl", "jvm_maven_import_external")
+
+jvm_maven_import_external(
+    name = "junit",
+    artifact = "junit:junit:4.12",
```
Comment (tpasternak) [inline examples/java/greetings_project/WORKSPACE:5]: Can we use the newest version (5.9.1)? Or at least 4.13.2, this one contains CVE

File: examples/java/greetings_project/WORKSPACE [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    artifact_sha256 = "59721f0805e223d84b90677887d9ff567dc534d7c502ca903c0c2b17f05c116a",
+    licenses = ["notice"],  # Common Public License 1.0
+    server_urls = ["https://repo1.maven.org/maven2"],
+)
+
+jvm_maven_import_external(
+    name = "truth",
+    artifact = "com.google.truth:truth:0.42",
```
Comment (tpasternak) [inline examples/java/greetings_project/WORKSPACE:13]: Most recent one is 1.1.3

File: examples/java/greetings_project/WORKSPACE [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    artifact_sha256 = "dd652bdf0c4427c59848ac0340fd6b6d20c2cbfaa3c569a8366604dbcda5214c",
+    licenses = ["notice"],  # Apache 2.0
+    server_urls = ["https://repo1.maven.org/maven2"],
+)
+
+jvm_maven_import_external(
+    name = "diffutils",
+    artifact = "com.googlecode.java-diff-utils:diffutils:1.2.1",
```
Comment (tpasternak) [inline examples/java/greetings_project/WORKSPACE:21]: This was last updated in 2011 https://mvnrepository.com/artifact/com.googlecode.java-diff-utils/diffutils/1.2.1.

Could we just compare elements one-by-one?

**Review by tpasternak — CHANGES_REQUESTED** [review]

File: examples/java/greetings_project/WORKSPACE (hunk 1 above)
Comment (mai93) [reply examples/java/greetings_project/WORKSPACE]: I updated this and removed the other external libraries.

**Review by tpasternak — APPROVED** [review]

**Pull request merged**